x-pack/plugins/search_indices/public/plugin.ts

```diff
@@ -89,8 +89,13 @@ export class SearchIndicesPlugin
     docLinks.setDocLinks(core.docLinks.links);
     if (this.pluginEnabled) {
       indexManagement?.extensionsService.setIndexDetailsPageRoute({
-        renderRoute: (indexName) => {
-          return `/app/elasticsearch/indices/index_details/${indexName}`;
+        renderRoute: (indexName, detailsTabId) => {
+          const route = `/app/elasticsearch/indices/index_details/${indexName}`;
+          const tabIds: string[] = Object.values(SearchIndexDetailsTabs);
```

Comment:
I think we should do this in `routes.ts` and export it as a const instead of doing this every function call.

```
export enum SearchIndexDetailsTabs {
  DATA = 'data',
  MAPPINGS = 'mappings',
  SETTINGS = 'settings',
}
export const SearchIndexDetailsTabValues = Object.values(SearchIndexDetailsTabs);
```

Then you can just import `SearchIndexDetailsTabValues` here.

Reply:
updated in e9cdedbc3622cf053d6623a42118a847b047fff9